tests/e2e/support/objects/app-files/spaces/index.ts

```diff
@@ -108,4 +111,12 @@ export class Spaces {
     const { id } = this.#spacesEnvironment.getSpace({ key })
     await changeSpaceImage({ id, resource, page: this.#page })
   }
+
+  async createPublicLink(): Promise<void> {
+    const url = await createPublicLinkForSpace({ page: this.#page })
+    this.#linksEnvironment.createLink({
+      key: 'Link',
+      link: { name: 'Link', url }
```

Comment:
any reason why name ist static?

Reply:
Whenever we create a link from the UI level, the link gets created with the name `Link` by default no matter how many links we create for a single space. Link cannot be created with a specific name by using the web UI.  So, even if we pass the name dynamically from the feature file, the name won't be used actually on the UI level while automating the test. That's why I put a static name ` key: 'Link',` just to store the link information, and we do change this static name later on in the renaming step.

https://github.com/owncloud/web/blob/02a60335e006b8872e2eee7e19f427eb183eb51b/tests/e2e/support/objects/app-files/link/index.ts#L41-L52


Would you agree with this idea? Or should I make any changes? Any suggestions from your side?

Reply:
THX for clarification